packages/manager/modules/pci/src/components/hds/template.html

```diff
@@ -0,0 +1,21 @@
+<div class="mb-3">
+    <oui-checkbox
+        data-disabled="!$ctrl.hds.isValidSupportLevel || !$ctrl.hds.isValidForCertification || $ctrl.hds.isCertifiedProject"
+        data-model="$ctrl.model.hds"
+        class="mb-3"
+        data-on-change="$ctrl.onChange ? $ctrl.onChange(modelValue) : null"
```

Comment:
Following @marie-j comment here https://github.com/ovh/manager/pull/4336/files#r565087164, the methods binding type for callbacks should be `&`. This way you could get rid of `null` check

Reply:
Fixed